tailwind.config.js

```diff
@@ -27,7 +70,11 @@ module.exports = {
   theme: {
     colors: {
       ...colors,
-      primary: getColors(colorPrimary, [100, 200, 300, 400, 500, 600, 700, 800, 900]),
+      primary: getColors(colorPrimary, [100, 200, 300, 400, 500, 600, 700, 800, 900]), 
```

Comment:
as I see `getColors` only copies the object, so I think that we can change it to the:

```suggestion
      primary: colorPrimary,
```